great_expectations/agent/run.py

```diff
@@ -1,4 +1,4 @@
-from great_expectations.agent import GXAgent
+from great_expectations_cloud.agent import GXAgent
 
 
 def run_agent():
```

Comment:
Going to delete this script from the `cloud` repo

Reply:
Why?